### ISSUE TITLE
[ iOS ] Using .env value from sh Run Script in Build Phases

### ISSUE THREAD

**Issue by JohnDot42** [react-native-config/react-native-config#310]
Hello there,

I'm trying to access value from run script and i did'nt find any solution.
I tried to use it from the plist file, but as it's preprocessed i can't access to it.

Does anyone has a solution ?
Thanks.

**Comment by cesarm16** [comment]
any update on this?

**Comment by luispuig** [comment]
We are running on the same issue. We need to access the variable from the run script but the plist file seems not to be already processed.

Can anybody help?

**Comment by reimertz** [comment]
Same here. did you find a solution to these issues @luispuig, @cesarm16, @JohnDot42?

**Comment by luispuig** [comment]
Nope, I found another way to solve my problem, and I didn't get into this anymore.

Good luck and let us know if you find the way.

**Comment by alishah730** [comment]
@luispuig  could you please help how you solved ? issue is still open for an year .

**Comment by luispuig** [comment]
My solution was completely unrelated to this. I found a way to do it without the need of accessing the env variable. Sorry for not being able to help.

**Comment by pedro** [comment]
Hey folks,

You should be able to run a script to parse `.env` and export that to the environment. There's [Ruby's dotenv](https://github.com/bkeepers/dotenv), [forego](https://github.com/ddollar/forego) and many others.

For instance, using dotenv:

```
$ echo $FOO

$ cat .env
FOO=bar

$ dotenv 'echo $FOO'
bar
```

I don't have a handy example here, but the general idea then is that you'd:

- Wrap your script call with something that parses `.env`
- Make your script take values from environment variables
